spec/models/user_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Account::User, type: :model do
+  context 'validation tests' do
+    let!(:user) { build(:user) }
+
+    describe 'factory' do
+      it { is_expected.to be_truthy }
+    end
+
+    describe '#first_name' do
+      it { is_expected.to validate_presence_of(:first_name) }
+
+      it 'does not save the object with 1 letter' do
+        user.update(first_name: "A")
+        expect(user).not_to be_valid
+        expect(user.errors[:first_name]).to include("is too short (minimum is 2 characters)")
+      end
+    end
+
+    describe '#last_name' do
+      it { is_expected.to validate_presence_of(:last_name) }
+
+      it 'does not save the object with 50+ letters' do
```

Comment:
and you should rename the spec,  because it is not checking, if user is saved in database.